src/Modules/Internationalisation/Domain/Translation/Filter/TranslationFilterType.php

```diff
@@ -41,7 +41,8 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'choice_label' => fn (Application $application): string => ucfirst(
                     $application->trans($this->translator)
                 ),
-                'choice_filter' => fn (?Application $application): bool => (bool) $application?->hasEditableTranslations(),
+                'choice_filter' => fn (?Application $application): bool =>
```

Comment:
not a big fan of splitting a closure up like this, maybe change the parameter name to app so it can be on a single line?

Reply:
fixed 9df0c0003ee8f5dfaf74b0e9c3ec69705cc57927